packages/manager/modules/iplb/package.json

```diff
@@ -29,9 +29,10 @@
     "angular-ui-bootstrap": "1.3.3",
     "bootstrap": "~3.3.0",
     "chart.js": "chartjs/Chart.js#^2.0",
+    "oclazyload": "^1.1.0",
```

Comment:
I don't see the dependency added on the standalone app. It was already added maybe ?

Reply:
You're right, I should add it to standalone apps.

Reply:
Added https://github.com/ovh/manager/pull/2753/files#diff-fe47c69617bc122898e7796aca924f27R58